mailviews/previews.py

```diff
@@ -180,7 +186,7 @@ def detail_view(self, request):
 
         message = message_view.render_to_message()
         raw = message.message()
-        headers = SortedDict((header, maybe_decode_header(raw[header])) for header in self.headers)
+        headers = OrganizedDict((header, maybe_decode_header(raw[header])) for header in self.headers)
```

Comment:
I meant importing it as follows:

```py
try:
    from collections import OrderedDict
except ImportError:
    from django.utils.datastructures import SortedDict as OrderedDict
```

and simply using `OrderedDict` below. Anyways, not a big deal :)